src/screens/Login/types.tsx

```diff
@@ -0,0 +1,9 @@
+export interface LoginState {
```

Comment:
Normally we'll name interface with prefix `I` as convention: `ILoginState` for example.